pkg/configReader/configReader.go

```diff
@@ -108,7 +108,7 @@ func ReadEnvVarsIntoStruct(s any) (any, error) {
 
 	validate := validator.New()
 	if err := validate.Struct(s); err != nil {
-		log.Debug().Msgf("Validation error: %v", err)
+		log.Debug().Err(err).Msg("validation error")
```

Comment:
Normally an error should be handled only once (logging an error is also handling it), so as we return the error to the caller we can probably just omit this log message.

If we log the error here and return it, it is to be expected that it will appear in the log twice if the caller decides to log the error as well.

Same for other occurences

Reply:
There was a discussion internally within the asset team on how errors and logging should be handled, and the outcome was that existing messages and control flow/returns should be kept as-is. I will bring this question up again.

Reply:
After discussion I think the outcome is that this can be removed, as this provides no additional (structured) information, which is not already present in the error.